saleor/graphql/webhook/mutations.py

```diff
@@ -232,7 +233,10 @@ class Arguments:
         id = graphene.ID(required=True, description="ID of a webhook to delete.")
 
     class Meta:
-        description = "Deletes a webhook subscription."
+        description = (
+            "Deletes a webhook subscription. Deletion might fail due to "
+            "webhook delivery in progress. In this case deactivates webhook"
```

Comment:
Redacted it to be more descriptive. This is our docs so needs to be as clear as possible to the users. 

```suggestion
            "Delete a webhook. Before the deletion, the webhook is deactivated to "
            "pause any deliveries that are already scheduled. The deletion might fail "
            "if delivery is in progress. In such a case, the webhook is not deleted "
            "but remains deactivated."
```

Reply:
Will need schema regeneration after it's updated.